### ISSUE TITLE
Model training issue 

### ISSUE THREAD

**Issue by TenzinJinpa2019** [tech-srl/code2seq#79]
Hi,

I am trying to train the model using the default model and it seems like model is not taking the GPU (I am using google colab pro ). I am not able to resolved the issue, could you kindly guide me ! 

Thanks

**Comment by urialon** [comment]
Hi @TenzinJinpa2019 ,
Thank you for your interest in code2seq!

I think that you need to choose "GPU" in the "Runtime -> Change Runtime Type" menu in Colab (this is unrelated to code2seq).

Uri

**Comment by TenzinJinpa2019** [comment]
Hi @urialon 

Thank you for the advice and I did change the runtime to  GPU. Also I tried to reduced the batch size. 

Current configuration 

a) GPU (through Google Colab )
b) Batch size 128 (training) and 64(training)

But  google Colab keeping showing me  the messages " allocated GPU but not utilizing  it " .

however  If I  run the program without changing the Batch size in TPU  it work but I want to use the GPU (hoping that it will be faster ). Is there any other config that I may need to change. 


with regards 
Tenzin Jinpa
